definitions/infra-awselasticbeanstalkinstance/summary_metrics.yml

```diff
@@ -1,8 +1,3 @@
-providerAccountName:
```

Comment:
same as previous comment

Reply:
it's not a metric, should it still be in there?

Reply:
In the summary metric section, you can also specify tags which will be displayed in the explorer as one of the columns. 
https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#tag

Reply:
got it, I'll put them back!